db/migrate/20180423160430_create_words.rb

```diff
@@ -0,0 +1,9 @@
+class CreateWords < ActiveRecord::Migration[5.1]
+  def change
+    create_table :words do |t|
+      t.string :lemma
```

Comment:
What does lemma mean?

Reply:
Forgot to respond to this: lemma refers to the key or headword in a dictionary, glossary, lexicon, etc.